recipe/meta.yaml

```diff
@@ -87,7 +87,7 @@ requirements:
 {% set tests_to_skip = tests_to_skip + " or test_apply_parallel_rgb" %}  # [win]
 
 # skip known failure on PyPy
-{% set tests_to_skip = tests_to_skip + " or test_hough_ellipse_zero_angle or test_hough_ellipse_non_zero" %}  #  [python_impl == 'pypy']
+{% set tests_to_skip = tests_to_skip + " or test_hough_ellipse_zero_angle or test_hough_ellipse_non_zero" %}  # [python_impl == 'pypy']
```

Comment:
The test is only failing on windows pypy, right?  So I think this needs an `and win`.

Reply:
Yeah, it was really only one of the PyPy cases. I can make it more selective as you suggest